spec/markup_helper_spec.rb

```diff
@@ -29,4 +27,27 @@
       [t_ident(:c), t_times, t_ident(:d)],
     ])
   end
+
+  it "can parse expressions" do
+    ast = Liquider::Ast::BinOpNode.new(
+      :+,
+      NumberNode.new(5),
+      NumberNode.new(4),
+    )
+    expect(parse_expression('5 + 4')).to eq(ast)
+  end
+
+  it "can parse arguments" do
+    ast = ArgListNode.new(
+      [
+        StringNode.new("titi"),
+        StringNode.new("toto"),
+      ],
+      OptionPairNode.new(
+        "tu",
+        StringNode.new("tu"),
+      )
+    )
+    expect(parse_arguments('"titi", "toto", tu: "tu"')).to eq(ast)
```

Comment:
@sgnr this test will fail because the strings get parsed as 

`StringNode.new('"titi"')`